src/plone/api/content.py

```diff
@@ -261,7 +261,10 @@ def copy(source=None, target=None, id=None, safe_id=False):
 
     new_id = copy_info[0]['new_id']
     if id:
```

Comment:
I always get gray hair if someone uses Python built-ins as variables, but that's not your fault here.